client/packages/invoices/src/Prescriptions/api/operations.graphql

```diff
@@ -50,6 +50,14 @@ fragment PrescriptionRow on InvoiceNode {
     firstName
     lastName
   }
+
+  currency {
```

Comment:
offroads? where did this lot come from?

Reply:
Offroads, the currency changes in outbounds broke prescriptions (: